### PR TITLE
Added Cypress with initial tests

### PR DESCRIPTION
**Pull request by Evan-CTL**
This is the very least I can do for now that is passing. Github actions is still struggling with the commented commands. You don't have to merge it, but I'm putting it up for review, just so you can look it over.

Side Note:
I would recommend adding ids to important elements, or restructure labels for inputs so that the inputs are easier to get in cypress.

### CODE REVIEW

File: metricsmentor/settings_shared.py [hunk 1]
```diff
@@ -72,3 +72,15 @@
 }
 
 COURSEAFFILS_COURSESTRING_MAPPER = CourseStringMapper
+
+DATABASES = {
```
Comment (ndittren) [inline metricsmentor/settings_shared.py:76]: You want this to be the case only if the integrationserver is running. Like so: https://github.com/ccnmtl/logiclearner/blob/main/logiclearner/settings_shared.py#L75

File: package.json [hunk 2]
```diff
@@ -6,6 +6,10 @@
   "scripts": {
     "build:dev": "webpack --node-env development",
     "build:prod": "webpack --config webpack.prod.js",
+    "cypress:run": "cypress run",
+    "cypress:headed": "cypress run --headed",
```
Comment (ndittren) [inline package.json:10]: For CI/CD purposes we would normally run these tests headlessly. We should leave the headed target here but for github actions, we should just run cypress:run

**Comment by ndittren** [comment]
> This is the very least I can do for now that is passing. Github actions is still struggling with the commented commands. You don't have to merge it, but I'm putting it up for review, just so you can look it over.
> 
> Side Note: I would recommend adding ids to important elements, or restructure labels for inputs so that the inputs are easier to get in cypress.

Some of these will get ID's once Zarina is goes through the code. But for elements you want to get in cypress you usually use this: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

File: package.json (hunk 2 above)
Comment (Evan-CTL) [reply package.json:10]: I had that originally. This was a proposed work-around that I had tried. Apparently it had worked for some people. I'll put it back

**Pull request closed without merging**